plugins/modules/activation_key.py

```diff
@@ -62,12 +62,15 @@
         description:
           - Pool ID of the Subscription to be added.
           - Mutually exclusive with I(name) and I(upstream_pool_id).
+          - Also named I(Candlepin Id) in the CSV export of the subscriptions,
+          - it is as well the UUID as output by I(hammer subscription list).
         type: str
         required: false
       upstream_pool_id:
         description:
           - Upstream Pool ID of the Subscription to be added.
           - Mutually exclusive with I(name) and I(pool_id).
+          - Also named I(Master Pools) at access.redhat.com.
```

Comment:
```suggestion
          - Also named C(Master Pools) in the Red Hat Portal.
```

`I()` is for option names, so I'd avoid that here. See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

And I think we should call the portal by name, not url (and if url, then `U(https://access.redhat.com)`)